python/paddle/incubate/hapi/utils.py

```diff
@@ -12,13 +12,149 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import inspect
 import numpy as np
 
+from collections import OrderedDict
 from paddle import fluid
 from paddle.fluid.framework import Variable
 from paddle.fluid.executor import global_scope
 
+__all__ = ['uncombined_weight_to_state_dict']
+
+
+def uncombined_weight_to_state_dict(weight_dir):
+    """
+    Convert uncombined weight to state_dict
```

Comment:
Maybe give more details for what are `uncombined weight`

Reply:
done, thanks.